src/ansys/fluent/core/launcher/pyfluent_enums.py

```diff
@@ -167,7 +167,7 @@ def _get_fluent_launch_mode(start_container, container_dict, scheduler_options):
         if check_docker_support():
```

Comment:
We can do something like this:
```
def check_docker_support():
    """Check whether Python Docker SDK is supported by the current system."""
    import docker
    docker.from_env()
```
and here
```
try:
    check_docker_support():
    ...
except DockerException as ex:
    raise DockerContainerLaunchNotSupported() from ex
```
so the original docker exception message will be attached.

I think checking whether docker is correctly setup here before processing further is fine.

Reply:
I was originally thinking exactly that in https://github.com/ansys/pyfluent/pull/2917#discussion_r1629472107, but then I realized that `check_docker_support()` seems to just be adding unnecessary complexity to the code, and not doing anything particularly useful. 

If we just remove `check_docker_support()` from the code, the same exception it was catching will naturally occur when we call `docker_client = docker.from_env()`.

In addition, `docker.from_env()` seems like a relatively heavy weight call to use and discard its return, just to check if Docker is supported in the current system, when I believe the Docker error handling should already handle that issue relatively gracefully without PyFluent interfering

Anyway, this would be a small optimization and code cleanup, what you suggest is fine by me too

Reply:
Thanks, yes, it is better to simply propagate the exception from fluent_container.py and remove `check_docker_support()`. (I overlooked that `docker.from_env()` returns the docker client.)

Reply:
Done. Thank you.